pkg/configuration/configuration.go

```diff
@@ -222,11 +223,27 @@ func (ev *extendedViper) Set(key string, value interface{}) {
 	}
 }
 
+func bindEnvSnykErr(key string, err error) snyk_errors.Error {
+	return snyk_errors.Error{
+		Title:          "Unable to bind env",
+		Classification: "ACTIONABLE",
+		Level:          "warning",
+		Detail:         fmt.Sprintf("Unable to bind env: %v. Reason: %v", key, err),
+	}
+}
+
 func (ev *extendedViper) get(key string) interface{} {
 	ev.mutex.Lock()
 	defer ev.mutex.Unlock()
 
-	// try to lookup given key
+	// ensure we only get Snyk related keys (prefixed with SNYK_)
+	if ev.GetKeyType(key) == EnvVarKeyType {
+		err := ev.viper.BindEnv(key, strings.ToUpper(key))
+		if err != nil {
+			return bindEnvSnykErr(key, err)
```

Comment:
Issue: I'm a bit concerned about this error handling here. Returning an error instead of a value is misleading.

Reply:
Looking at the error cases in `BindEnv()` it can be avoided or ignored since the only error case is, that there is no input. 

But we should definitely not stop further processing here if something fails.

Reply:
> Issue: I'm a bit concerned about this error handling here. Returning an error instead of a value is misleading.

Yeh, this is abusing the `interface{}` return type a bit here, we should be consistent in what we return.

> Looking at the error cases in BindEnv() it can be avoided or ignored since the only error case is, that there is no input.

We can't be sure this will always be the case, would logging the error and returning nil be a good compromise?